composer.json

```diff
@@ -6,7 +6,9 @@
 	"license": "BSD-3-Clause",
 	"require": {
 		"silverstripe/framework": "~3.1",
-		"silverstripe/cms": "~3.1"
```

Comment:
You could throw a PSR-4 autoload declaration in here if you want. :p You've already namespaced everything right?